libs/ui/packages/react/src/pre-ldls/components/CryptoIcon/CryptoIcon.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { type Meta, type StoryObj } from "@storybook/react";
+import { CryptoIcon } from "./CryptoIcon";
+
+const meta: Meta<typeof CryptoIcon> = {
+  component: CryptoIcon,
+  title: "PreLdls/Components/CryptoIcon",
+  tags: ["autodocs"],
+  args: {
+    ledgerId: "bitcoin",
+    ticker: "BTC",
+    size: "56px",
+  },
```

Comment:
There is also a network prop could add it the story args please (maybe just an empty string so we can use the control to set it).